### DISCUSSION TITLE
ExecuteAsync parameters

### DISCUSSION THREAD

**Discussion by AhmedJoum** [paillave/Etl.Net#404] [Q&A] [answered]
What is the `string config` parameter in ExecuteAsync, and what to pass on it,
Note: I'm using ETL.net to load from an SQL server to another SQL server with ef core in a WebApi project

**Comment by paillave** [comment, marked as answer]
It is documented here in the detailled principle:

https://paillave.github.io/Etl.Net/docs/quickstart/principle#detailed-principle

> The runtime triggers the process by raising a single event containing parameters of the ETL process into the trigger stream

And here in "from trigger stream"

https://paillave.github.io/Etl.Net/docs/recipes/useExternalData#from-trigger-stream
